notebook/nbconvert/handlers.py

```diff
@@ -103,6 +105,7 @@ def get(self, format, path):
                 }
             )
         except Exception as e:
+            self.log.exception("nbconvert failed: %s", e)
```

Comment:
OK. I find it a bit odd that you have done this in two different ways within one PR, but that's not important.